esmvalcore/preprocessor/_volume.py

```diff
@@ -293,23 +348,7 @@ def depth_integration(cube):
         collapsed cube.
     """
     # ####
```

Comment:
What's the purpose of this line? (I know that you didn't introduce it here.) If nobody knows, let's remove it?